src/backend/joanie/core/models/products.py

```diff
@@ -69,6 +70,9 @@
 session.mount("https://", adapter)
 
 
+TOTAL_DECIMAL_PLACES_ACCEPTED = Decimal(10) ** -2
```

Comment:
Isn't more explicit like that ?
```suggestion
TOTAL_DECIMAL_PLACES_ACCEPTED = Decimal('0.01')
```